doc/running.rst

```diff
@@ -119,3 +119,42 @@ A typical script, therefore, could look like the following. For this example, we
    nest.Install("nestmlmodule")
    # ...
    nest.Simulate(400.)
+
+Running NESTML with custom templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+NESTML generates model-specific C++ code for the NEST simulator using a set of Jinja templates. By default, NESTML uses the templates in the directory ``pynestml/codegeneration/resources_nest/point_neuron``. For more information on code generation using templates, see :ref:`Section 3.1: AST Transformations and Code Generation`.
+
+The default directory can be changed through ``--codegen_opts`` by providing a path to the custom templates as an option in a JSON file.
+
+.. code-block:: bash
+   nestml --input_path models/iaf_psc_exp.nestml --codegen_opts /home/nest/work/codegen_options.json
+An example ``codegen_options.json`` file is as follows:
+
+.. code-block:: json
```

Comment:
This seems inconsistent with the proposed format, could you update this? Also some line breaks seem to be missing from the rst file.

Reply:
My bad. I have updated it now.